hw1/countline.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
You may not change the question :)

Reply:
Done. Thanks. I will be more careful.

Reply:
Thanks.  To make the history look clearer, why don't you squash or avoid the merge back?